File: src/main/java/org/taktik/icure/dto/filter/healthelement/HealthElementByHcPartyTagCodeFilter.java
package org.taktik.icure.dto.filter.healthelement;

import org.taktik.icure.dto.filter.Filter;
import org.taktik.icure.entities.HealthElement;

public interface HealthElementByHcPartyTagCodeFilter extends Filter<String, HealthElement> {
	String getHealthCarePartyId();

	String getCodeType();

	String getCodeNumber();

	String getTagType();

	String getTagCode();

	Integer getStatus();
}

File: src/main/kotlin/org/taktik/icure/domain/filter/healthelement/HealthElementByHcPartyTagCodeFilter.kt
package org.taktik.icure.domain.filter.healthelement

import org.taktik.icure.domain.filter.Filter
import org.taktik.icure.entities.HealthElement

interface HealthElementByHcPartyTagCodeFilter : Filter<String, HealthElement> {
    val healthCarePartyId: String?
    val codeType: String?
    val codeNumber: String?
    val tagType: String?
    val tagCode: String?
    val status: Int?
}
